internal/commands/init.go

```diff
@@ -15,11 +15,11 @@ var (
 
 func initCmd(dockerCli command.Cli) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "init APP_NAME [--compose-file COMPOSE_FILE] [OPTIONS]",
-		Short:   "Initialize Docker Application definition",
-		Long:    `Start building a Docker Application package.`,
-		Example: `$ docker app init myapp`,
-		Args:    cli.ExactArgs(1),
+		Use:   "init [OPTIONS] APP_NAME",
```

Comment:
@carolinebriaud with second thoughts, `APP_NAME` feels wrong... Should it be `APP_DEFINITION` ? We don't use `APP_NAME` anywhere else.

Reply:
Right! Modified.